### PR TITLE
Frontend/typescrypt

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (MonsieurLanza) [inline CHANGELOG.md:8]: Lots of unrelated changes, but no `typescript` ? :stuck_out_tongue_winking_eye:

File: src/frontend/web_application/webpack/config.js [hunk 2]
```diff
@@ -7,6 +7,49 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
...
 const clientOptions = require('../config/client.default.js');
 
+const isDev = process.env.NODE_ENV === 'development';
+
+const configureSrcTsLoader = ({ isNode } = { isNode: false }) => {
+  return {
+    module: {
+      rules: [
+        {
+          test: /(?<!\.worker)\.(j|t)sx?$/,
```
Comment (MonsieurLanza) [inline src/frontend/web_application/webpack/config.js:17]: Hence, no file renaming ?

File: src/frontend/web_application/src/scenes/RemoteIdentitySettings/presenter.jsx [hunk 3]
```diff
@@ -43,15 +43,18 @@ class RemoteIdentitySettings extends Component {
 
   handleError = (error) => {
     // TODO : better error handling
-    console.log(error);
+    // console.log(error);
```
Comment (MonsieurLanza) [inline src/frontend/web_application/src/scenes/RemoteIdentitySettings/presenter.jsx:46]: :stuck_out_tongue:

File: CHANGELOG.md (hunk 1 above)
Comment (iamdey) [reply CHANGELOG.md:8]: lol no, it prepares for typescript: the only thing to do now is to rename file from js to ts.

Almost all the changes (removing `@lingui/macro`) is for typescript compat.

And well typescript stuff is not much useful in changelog imo

File: src/frontend/web_application/webpack/config.js (hunk 2 above)
Comment (iamdey) [reply src/frontend/web_application/webpack/config.js:17]: I have a lot of diff that is already conflicts with recent changes (prettier) I prefer keep conventionnal file naming

**Pull request merged**